js/validate.js

```diff
@@ -0,0 +1,20 @@
+const form = document.querySelector('form');
+const email = document.getElementById('email');
+const emailErrorContainer = document.querySelector('span.error');
+
+email.addEventListener('input', () => {
+  if (emailErrorContainer.classList.contains('active')) {
+    emailErrorContainer.classList.remove('active');
+  }
+});
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (email.value !== email.value.toLowerCase()) {
+    emailErrorContainer.textContent = 'You are required to use only lowercase letters in your email.';
+    emailErrorContainer.classList.add('active');
```

Comment:
[OPTIONAL] Well done 👏🏽 on displaying the error message, it really helps the user understand what's going on. I'd suggest improving it even further 💯 to use the `setTimeout` method to make the message disappear after an amount of time `5s` is correct because that's enough time for the user to read it. It would go like that:
```
    setTimeout(() => {
      emailErrorContainer.classList.remove('active');
    }, 5000)
 ```

Reply:
Thank you for your important suggestion @didierganthier, 🙏 
I am grateful. 👍